### DISCUSSION TITLE
Penpot assets for Skeleton

### DISCUSSION THREAD

**Discussion by notramo** [skeletonlabs/skeleton#2797] [Ideas]
[Penpot](https://penpot.app/) is an open-source Figma alternative, with some distinctive features. It's designed from the ground up for designer - developer collaboration, that's why many developers prefer it over Figma, which is mainly for designers.

There is a Figma exporter plugin to transfer projects from Figma to Penpot, but native support would be better.
https://github.com/penpot/penpot-exporter-figma-plugin

**Comment by endigo9740** [comment]
Hey @notramo, yeah we're aware of Penpot. Currently we have a set of Figma assets provided by community members Micha and Bohdan for Skeleton v2. This is free and public and you can access it here:
https://www.skeleton.dev/docs/figma

They did such a great job with this, Skeleton Labs is working with them in an official capacity to generate our Figma assets for Skeleton v3. They won't be available until close to launch, but I call tell you they've knocked it out of the park. There's a ton of variations and the quality is top notch. Here's a quick preview.

![image](https://github.com/user-attachments/assets/ab252fdc-de5d-440b-b205-9c5bb31fd06f)

![image](https://github.com/user-attachments/assets/97a7e6f7-1a94-4968-8185-8b18648caeff)

Note that there will be a reasonable price asked for these, comparable to what other frameworks charge. But we may also end up bundling with other useful assets, such as templates, etc.

That said, this has been a fairly major effort for a company and team our size so I wouldn't expect a direct Penpot port anytime soon. However, if an exporter exists perhaps it's something that would be in your control?

Either way I appreciate the request and definitely recommend keeping an eye out for v3 progress. We'll be sharing more about this in the future.
